services/app-api/src/authn/thirdPartyAuthn.ts

```diff
@@ -17,8 +18,20 @@ export async function userFromThirdPartyAuthorizer(
 
     const serviceName = 'third-party-authorizer'
     initTracer(serviceName, otelCollectorURL)
-
+    const allowedIpAddresses = process.env.ALLOWED_IP_ADDRESSES
+    if (allowedIpAddresses === undefined || allowedIpAddresses === '') {
+        throw new Error(
+            'Configuration Error: ALLOWED_IP_ADDRESSES is required to run app-api.'
+        )
+    }
     try {
+        const ipAddressIsValid = allowedIpAddresses.includes(ipAddress)
```

Comment:
This is a little strange to me, using string matching against potentially an array of ip addresses, but I think it works fine. honestly it's nice that it's robust against spaces and other stuff being in the array.